chapters/3-measurement.tex

```diff
@@ -37,125 +39,210 @@
-or after doing a respondent listing before a survey.
-
-You also need to record how all datasets for each unit of observation
-will link or merge with each other as needed.
-This linking scheme is called a \textbf{data map}\sidenote{
-	\url{https://dimewiki.worldbank.org/data\_map} (TO BE CREATED)}.
-\index{data maps} 
-A data map is more than just a list of datasets.
-Its purpose is to specify the characteristics and linkages of those datasets.
-To link properly, the master datasets must include fully unique ID variables.\sidenote{
-	\url{https://dimewiki.worldbank.org/ID\_Variable\_Properties}}
-The master datasets should indicate whether datasets should be merge one-to-one,
-for example, merging baseline data and endline data that use the same unit of observation,
+list all observations your project ever encounter
+are the authoritative source for all research data
+such as ID values, sampling and treatment status, etc.
+\textbf{Data flow charts}\sidenote{
+	\url{https://dimewiki.worldbank.org/Data\_Flow\_Chart}}
+list all datasets that are needed to create each analysis dataset,
+and how they should be created by merging, appending 
+or in any other way link different datasets.
+
+\subsection{Creating a data plan}
+
+The components of the data plan is the best tools a research team has for
+the lead researchers to communicate their vision for the data work, 
+and for the research assistants to communicate their understanding of that vision.
+While the time to create the data plan is before any data has been acquired,
+it should be a dynamic documentation, 
+that you should keep up to date as the project evolves.
+
+To create a data plan according to DIME's template, first,
+start by creating a \textbf{data linkage table} by listing 
+all the datasets you know that you will use in a spreadsheet.
+If one source of data will results in two different dataset, 
+then list each dataset on a their own row.
+Then for each dataset list the \textbf{unit of observation}\sidenote{
+	\url{https://dimewiki.worldbank.org/Unit\_of\_Observation}},
+and the name of the main ID variable for that unit of observation.
+Your project should only have one main ID variable per unit of observation,
+called the project ID. 
+When you list a dataset in the data linkage table --
+which should be done before that dataset is acquired --
+you should always make sure that the dataset will
+be fully identified by the project ID, 
+or make a plan for how 
+the new dataset will be linked to the project ID.
+It is very labor expensive to work with a dataset that
+do not have an unambiguous way to link to the project ID, 
+and it is a big source of error.
+
+The data linkage table should indicate whether 
+datasets should be merge one-to-one, for example, 
+merging baseline data and endline data that use the same unit of observation,
 or whether two datasets should be merged many-to-one, 
 for example, school administrative data merged with student data.
```

Comment:
```suggestion
The data linkage table should indicate whether 
datasets will merge one-to-one, 
such as merging survey data and census data for individuals,
or whether two datasets will merge many-to-one, 
such as school-level administrative data merged with student data.
```

First one is rather an append in most cases; reduced use of commas for clarity

Reply:
Can we use "may be merged"? Not all datasets that are possible to merge will be merge.